Week3/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>A Button Should be Here</title>
+  <style>
+    .button {
```

Comment:
https://www.spotify.com/nl/

Make the button look like spotify.

Reply:
Take a look